### ISSUE TITLE
OIDC with Citrix Netscaler

### ISSUE THREAD

**Issue by Fabe0303** [nextcloud/user_oidc#837]
Hi Folks,

We are trying to use our Citrix Netscaler as IdP for our Nextcloud-Server. At the moment our Users use LDAP which is configured on our Nextcloud to authenticate. Our LDAP Login Attributes for the Internal Username is set to the default setting which is the ObjectUUID.

Now we try to get the ObjectUUID from our OIDC Netscaler Server, but we think the OIDC App does not ask for a Claimed Controlling  IDTOKEN(CC IDToken) which can be delivered the ObjectUUID to the Nextcloud Server to map the backend User.

Does the OIDC Nextcloud Support CC IDToken?

**Comment by edward-ly** [comment]
I don't believe we support anything that is outside of the [OIDC specification](https://openid.net/developers/specs/), but feel free to correct me if I'm wrong. I can't find any information regarding whether Netscaler (ADC?) supports OIDC either.
